### DISCUSSION TITLE
Host and Subnet Name Mapping

### DISCUSSION THREAD

**Discussion by medtemo** [Security-Onion-Solutions/securityonion#8135] [Unsupported Versions] [answered]
Hello OnionMasters!

I have quick question. 
Does it possible for Security Onion to be able to map hosts and subnet names and visible in SOC or Kibana as additional field. 
e.g. 

10.0.0.0/8     ---- Corporate vlan
10.10.10.10  ---- DNS Server

**Comment by dougburks** [comment, marked as answer]
You could add additional fields during ingest:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing

**Comment by xfaith** [comment]
Could this be done using like a CSV file?  Dont have DNS/AD, but can build out a rough ip/hostname file for the main systems.  Trying to figure out if that works in parsing.  I thought it would be done in logstash, but based off above looks like it could possibly be done in elastic?

**Reply by dougburks** [reply]
Yes, this can be done in Elasticsearch ingest using the enrich processor and data loaded from a CSV file. Here's an example:
https://alexmarquardt.com/enriching-elasticsearch-documents-with-data-from-csv-files/
